config/meson.build

```diff
@@ -9,3 +9,15 @@ install_data(
   'agent/agent.conf',
   install_dir :  get_option('sysconfdir') / 'hirte'
 )
+
+install_data(
+  'agent/hirte-default.conf',
+  install_dir : '/usr/share/hirte/agent'
```

Comment:
I'd use `/usr/share/hirte-agent/config/` directory for the default config file, hirte-agent and hirte are independent RPM packages, so it would be easier to keep their `/usr/share` content separate

Reply:
Done